### DISCUSSION TITLE
Comic Getter moving forward

### DISCUSSION THREAD

**Discussion by magnuschase293** [magnuschase293/comic_getter#23] [Announcements]
Hi,
I enabled this section to receive feedback, suggestions and enhancements to Comic Getter and leave Issues exclusively for bug related comments. 

By the way, I would like to announce that later this year I will be releasing Cartoon Getter to download animated movies, anime and cartoons from [wcostream.com](https://www.wcostream.com/).

**Comment by StrangeDOS** [comment]
Awesome.  I look forward to testing it.
